scripts/install2

```diff
@@ -112,220 +179,50 @@ if [[ -e '/etc/NetworkManager/NetworkManager.conf' ]]; then
-        else
-            echo -e "\e[1;32mPrometheus Server will be installed\e[0m"
-            WITH_PROMETHEUS_SERVER=true
-        fi
-    fi
-else
-    # Skip because Prometheus Node Exporter is not installed
-    WITH_PROMETHEUS_SERVER=false
+    askModule "WITH_PROMETHEUS_SERVER" "Prometheus Server"
 fi
 if [ "$WITH_PROMETHEUS_SERVER" == true ]; then
-    if [ -z "$WITH_GRAFANA" -o "$WITH_GRAFANA" != "true" -a "$WITH_GRAFANA" != "false" ]; then
-        read -p "Install Grafana (Y/n)? " -n 1 -r
-        echo ""
-        if [[ $REPLY =~ ^[Nn]$ ]]; then
-            echo -e "\e[1;31mGrafana installation will be skipped\e[0m"
-            WITH_GRAFANA=false
-        else
-            echo -e "\e[1;32mGrafana will be installed\e[0m"
-            WITH_GRAFANA=true
-        fi
-    fi
-else
-    # Skip because Prometheus Server is not installed
-    WITH_GRAFANA=false
-fi
-if [ -z "$WITH_H_DNS" -o "$WITH_H_DNS" != "true" -a "$WITH_H_DNS" != "false" ]; then
-    read -p "Configure Hyperboria-compatible Domain Name Servers (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mDNS settings will be skipped\e[0m"
-        WITH_H_DNS=false
-    else
-        echo -e "\e[1;32mDNS settings will be configured\e[0m"
-        WITH_H_DNS=true
-    fi
+    askModule "WITH_GRAFANA" "Grafana"
 fi
-if [ -z "$WITH_H_NTP" -o "$WITH_H_NTP" != "true" -a "$WITH_H_NTP" != "false" ]; then
-    read -p "Configure Hyperboria-compatible Network Time Protocol server (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mNTP settings will be skipped\e[0m"
-        WITH_H_NTP=false
-    else
-        echo -e "\e[1;32mNTP settings will be configured\e[0m"
-        WITH_H_NTP=true
-    fi
-fi
-if [ -z "$WITH_EXTRA_TOOLS" -o "$WITH_EXTRA_TOOLS" != "true" -a "$WITH_EXTRA_TOOLS" != "false" ]; then
-    read -p "Install non-essential tools useful for network analysis (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mExtra tools will be skipped\e[0m"
-        WITH_EXTRA_TOOLS=false
-    else
-        echo -e "\e[1;32mExtra tools will be installed\e[0m"
-        WITH_EXTRA_TOOLS=true
-    fi
-fi
-if [[ "$SUPPORT_WATCHDOG" == "true" ]] && [ -z "$WITH_WATCHDOG" -o "$WITH_WATCHDOG" != "true" -a "$WITH_WATCHDOG" != "false" ]; then
-    read -p "Configure Hardware Watchdog (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mHardware Watchdog configuration will be skipped\e[0m"
-        WITH_WATCHDOG=false
-    else
-        echo -e "\e[1;32mHardware Watchdog will be configured\e[0m"
-        WITH_WATCHDOG=true
-    fi
+askModule "WITH_H_DNS" "Hyperboria Domain Name Servers"
+askModule "WITH_H_NTP" "Hyperboria Network Time Protocol Server"
+askModule "WITH_EXTRA_TOOLS" "Non-Essential Tools Useful for Network Analysis"
+if [ "$SUPPORT_WATCHDOG" == "true" ]; then
+    askModule "WITH_WATCHDOG" "Hardware Watchdog"
```

Comment:
You mentioned to me that it should always be installed. Should it be an option at all? And if it should still, can the description here point out how it is required?

Reply:
Watchdog resets the cpu when it decides its broken. I'd rather give the user the choice if they want to hand over that power or not.

Reply:
Alright. It might be good to point people towards the readme in this repo at the beginning of the install though, as all the modules are explained there.

Reply:
We need to do a better job of illustrating to module in general.  I think a separate page with module descriptions and usage